src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.html

```diff
@@ -5,6 +5,7 @@
  *--------------------------------------------------------------------------------------------*/
 -->
 <div style="width: 100%; height: 100%; display: flex; flex-flow: column">
+	<!-- <cell-toolbar-component *ngIf="cell.active"></cell-toolbar-component> -->
```

Comment:
Remove?

Reply:
Yes, I can remove them. They will be however be implemented in the release.

Reply:
That's fine - you can leave them in if you're going to be uncommenting in a later PR so it's easier to keep the context.